src/components/Calculator.js

```diff
@@ -1,36 +1,46 @@
-import React from 'react';
+import React, { useState } from 'react';
+import calculate from './logic/calculate';
 
-class Calculate extends React.PureComponent {
-  render() {
-    return (
-      <section>
-        <div className="Container">
-          <p className="Result" type="text">0</p>
-          <div className="CalcBtns">
-            <button className="Button Norm" type="submit">AC</button>
-            <button className="Button Norm" type="submit">+/-</button>
-            <button className="Button Norm" type="submit">%</button>
-            <button className="Button BoxColor" type="submit">÷</button>
-            <button className="Button Norm" type="submit">7</button>
-            <button className="Button Norm" type="submit">8</button>
-            <button className="Button Norm" type="submit">9</button>
-            <button className="Button BoxColor" type="submit">x</button>
-            <button className="Button Norm" type="submit">4</button>
-            <button className="Button Norm" type="submit">5</button>
-            <button className="Button Norm" type="submit">6</button>
-            <button className="Button BoxColor" type="submit">-</button>
-            <button className="Button Norm" type="submit">1</button>
-            <button className="Button Norm" type="submit">2</button>
-            <button className="Button Norm" type="submit">3</button>
-            <button className="Button BoxColor" type="submit">+</button>
-            <button className="Zero" type="submit">0</button>
-            <button className="Button Norm" type="submit">.</button>
-            <button className="Button BoxColor" type="submit">=</button>
-          </div>
+function Calculate() {
+  const [state, setter] = useState({ total: 0, next: null, operation: null });
```

Comment:
:red_circle: Great implementation of the state :clap: :+1:  But remember the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_events.md) states that you should also use props :arrow_down: 
![Screenshot from 2023-01-27 13-48-19](https://user-images.githubusercontent.com/26736582/215080640-2ad131ad-986d-46a4-a94f-b4dcb1b4f82c.png)

Kindly make sure you use props somewhere in your project.  What do you think? :hugs: 
You can for example make your result box a component which takes the total, operation and next as props
-  [ ] Create first a functional component  
` const Result = ({total, operation, next}) =>  <div>{total} {operation} {next}</div>`
then you can replace :arrow_down: 
```
<div className="Result">
          {total}
          {operation}
          {next}
        </div>
```
with this :arrow_down: 

```
<Resut
  operation = {operation}
  total = {total}
 next = {next}
/>
```